src/redprl/judgment.fun

```diff
@@ -27,6 +27,7 @@ struct
            in
              (([],[]), tau)
            end
+       | MATCH_RECORD (lbl, _) => (([],[]), RedPrlSortData.EXP)
```

Comment:
I can never remember which structure aliases are defined where 😦 

You could just define `structure S = RedPrlSortData` locally in this module, or you could try and see if something like that is already in scope.

BTW [off-topic], one of the frustrating things about ML modules is that when you ascribe a signature, even transparently, you can't access any datatype constructors that are defined in the module externally—unless you extend the signature, which results in a doubling of code.